### PR TITLE
Fix README errors

### PR DESCRIPTION
**Pull request by knihit**
*Issue #, if available:*

*Description of changes:* Updates to the README for kinesisdatastreams-gluejob to fix links and inaccuracies.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: codebuildgithubautobuildPro-QfXHf1ULVFb1
* Commit ID: ba8968a649d2c52da34a9b2f0084c1e2c7e50794
* Result: FAILED
* [Build Logs](https://idenu9lil3.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=6752c1cd-28f6-4984-ab96-a1011ff2bd91%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: codebuildgithubautobuildPro-QfXHf1ULVFb1
* Commit ID: 70a45b41f7d2bb1a6b11b85df18d0b6fed91d55c
* Result: SUCCEEDED
* [Build Logs](https://idenu9lil3.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=ec8b241a-745a-40f2-9825-ba5346759d5a%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: codebuildgithubautobuildPro-QfXHf1ULVFb1
* Commit ID: da4be4a7d6c632cba375cd695b585e55080046bc
* Result: SUCCEEDED
* [Build Logs](https://idenu9lil3.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=f6549f7c-0d1d-48d4-b591-595c481bcee3%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: codebuildgithubautobuildPro-QfXHf1ULVFb1
* Commit ID: 88bce470607942ad1376180545540319acff598a
* Result: SUCCEEDED
* [Build Logs](https://idenu9lil3.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=c83fdcc8-cc8b-4a96-83d6-09219a4ee74d%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: source/patterns/@aws-solutions-constructs/aws-kinesisstreams-gluejob/README.md [hunk 1]
```diff
@@ -85,30 +85,30 @@ _Parameters_
...
+| kinesisStreamProps? | [`kinesis.StreamProps`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-kinesis.StreamProps.html)                | Optional user-provided props to override the default props for the Kinesis stream.                               |
+| existingStreamObj?  | [`kinesis.Stream`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-kinesis.Stream.html)                          | Existing instance of Kinesis Stream, if this is set then kinesisStreamProps is ignored.                          |
+| glueJobProps?       | [`cfnJob.CfnJobProps`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-glue.CfnJobProps.html)                    | User provided props to override the default props for the AWS Glue Job.                                          |
+| existingGlueJob?    | [`cfnJob.CfnJob`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-glue.CfnJob.html)                              | Existing Existing instance of AWS Glue Job, if this is set then glueJobProps is ignored.                         |
```
Comment (biffgaut) [inline source/patterns/@aws-solutions-constructs/aws-kinesisstreams-gluejob/README.md:93]: I'm getting a little lost in the green/red adds/deletions, but it looks like "Existing" is repeated here?

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: codebuildgithubautobuildPro-QfXHf1ULVFb1
* Commit ID: e0f22eb058f8259d1c5ad06670e0926644e1d98e
* Result: SUCCEEDED
* [Build Logs](https://idenu9lil3.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=9a2d5ad9-de02-4b59-99c8-2e6256d23a19%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: source/patterns/@aws-solutions-constructs/aws-kinesisstreams-gluejob/README.md (hunk 1 above)
Comment (knihit) [reply source/patterns/@aws-solutions-constructs/aws-kinesisstreams-gluejob/README.md]: Fixed.

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: codebuildgithubautobuildPro-QfXHf1ULVFb1
* Commit ID: 1240701ba3ac49f0fe07215e2848c701e4cf59e2
* Result: SUCCEEDED
* [Build Logs](https://idenu9lil3.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=a2da74bf-b4b7-44ec-a457-2857de584581%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by hnishar — APPROVED** [review]

**Pull request merged**